components/GroupSearchFilters/GroupSearchFilters.js

```diff
@@ -2,7 +2,8 @@ import PropTypes from 'prop-types';
 
 import { useModalDispatch, showModal } from 'providers/ModalProvider';
 import { useGroupFilters, resetValues } from 'providers/GroupFiltersProvider';
-import { Box, Button, Divider } from 'ui-kit';
+import { Box, Button, Divider, Icon } from 'ui-kit';
+import { rem } from 'ui-kit/_utils';
```

Comment:
You can do `import { Box, Button, Divider, Icon, utils } from 'ui-kit';` and then `utils.rem()`.

Reply:
That's good to know, I did this `_utils` import in another spot. Didn't bother to look to see if there was a more convenient import...